__brick__/.eslintrc.json

```diff
@@ -7,6 +7,11 @@
     "simple-import-sort"
   ],
   "rules": {
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/no-misused-promises": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-confusing-void-expression": "off",
+    "@typescript-eslint/no-floating-promises": "off",
```

Comment:
perché stiamo spegnendo questa rule? Qual è il caso d'uso in cui rema contro?

https://typescript-eslint.io/rules/no-floating-promises/